releases/release-announcement-template.md

```diff
@@ -0,0 +1,61 @@
+# Announcing `bindgen` $TODO_VERSION
+
+`bindgen` automatically generates Rust FFI bindings to C and C++ libraries.
+
+* [GitHub][]
+* [crates.io][]
+
+Upgrade to this release by updating your `Cargo.toml`:
+
+```toml
+bindgen = "$TODO_VERSION"
+```
+
+## Changelog
+
+### Added
+
+* TODO (or remove section if none)
+
+### Changed
+
+* TODO (or remove section if none)
+
+### Deprecated
+
+* TODO (or remove section if none)
+
+### Removed
+
+* TODO (or remove section if none)
+
+### Fixed
+
+* TODO (or remove section if none)
+
+### Security
+
+* TODO (or remove section if none)
+
+
+## Friends
+
+Thanks to everyone who contributed to this release!
+
+<insert the output of friends.sh here>
+
+## Contributing
+
+Want to join us? Check out our [CONTRIBUTING.md][contributing] and take a look
+at some of these issues:
+
+* [Issues labeled "easy"][easy]
+* [Issues labeled "less easy"][less-easy]
+* Still can't find something to work on? [Drop a comment here][looking]
+
```

Comment:
I would add some boilerplate text about non-code contributions (reporting bugs, docs, etc.) and maybe a direct link to file a new issue